### ISSUE TITLE
Locations in the SessionSeries *and* ScheduledSession

### ISSUE THREAD

**Issue by domfennell** [Playwaze/opendata#7]
Hi @JoshuaLevett,

We have noticed that the `ScheduledSession`s in your [ScheduledSessions feed](https://playwaze.com/OpenData/ScheduledSessions) all have a `location` or a `beta:affiliatedLocation`. The `SessionSeries` also have these in your [SessionSeries feed](https://playwaze.com/OpenData/SessionSeries). It is generally best practice to only include `location` (or `beta:affiliatedLocation`) data in the `SessionSeries`. In the OpenActive guidance [here](https://developer.openactive.io/publishing-data/data-feeds/types-of-feed), the examples generally include location data in the `SessionSeries`. It also states that it is possible for a `ScheduledSession` to include a location if it wishes to override the location of its `SessionSeries` parent.

Presently, we do not yet support having a `ScheduledSession`'s location override the `SessionSeries`'s location (though we may do in the future). For now, is it possible to just have `location/beta:affiliatedLocation` data in the `SessionSeries` and only include it in the `ScheduledSession` if it occurs at a different location than usual for some exceptional reason? We believe this makes it clearer that all the `ScheduledSessions` within the same `SessionSeries` happen in the same location.

We've also noticed a couple other small issues related to location:

- `name` and `address` are in the `GeoCoordinates` object as well as the `Place` object. In the modelling spec, these should not be present in `GeoCoordinates` (see [here](https://www.openactive.io/modelling-opportunity-data/#describing-geographic-location)). Could you remove these two fields from `GeoCoordinates` and keep them in `Place`?
- Some locations have a lat/lng but no address. We do not support showing sessions that have no address to users as a lat/lng with no address can be ambiguous. For example, it could include multiple locations. An example of this is https://playwaze.com/SessionSeries/ybn5byqvxnn9y. If the address data is just not known, then that's fine, but we wanted to flag it in case this is caused by a bug.

Best,

Dom

**Comment by charlieclarke5** [comment]
@JoshuaLevett please can you update me on this one? I know we only discussed back end of last week. Thanks.

**Comment by JoshuaLevett** [comment]
Apologies for the delay in getting round to clearing this off @domfennell.
We've removed location from `ScheduledSessions`, due to the way we now send them, we can guarantee that all `SchSe` within a `SeSe` will be in the same location.
We've also removed `Name` and `Address` from within `GeoCoordinates` (a case of filling in every field possible).

Finally The `SeSe` you reference was created before we properly managed venues/locations so it's data quality is a bit lower,no bugs involved this time!

Thanks for the feedback and all the best,

Josh
